pandas/tests/arrays/categorical/test_replace.py

```diff
@@ -39,34 +38,3 @@ def test_replace_categorical_series(to_replace, value, expected, flip_categories
 
     tm.assert_series_equal(expected, result, check_category_order=False)
     tm.assert_series_equal(expected, ser, check_category_order=False)
-
-
-@pytest.mark.parametrize(
-    "to_replace, value, result, expected_error_msg",
-    [
-        ("b", "c", ["a", "c"], "Categorical.categories are different"),
-        ("c", "d", ["a", "b"], None),
-        # https://github.com/pandas-dev/pandas/issues/33288
-        ("a", "a", ["a", "b"], None),
-        ("b", None, ["a", None], "Categorical.categories length are different"),
-    ],
-)
-def test_replace_categorical(to_replace, value, result, expected_error_msg):
-    # GH#26988
-    cat = Categorical(["a", "b"])
-    expected = Categorical(result)
-    with tm.assert_produces_warning(FutureWarning, match="Series.replace"):
-        # GH#44929 replace->_replace
-        result = cat.replace(to_replace, value)
```

Comment:
worth keeping this test and replacing this line with `result = Series(cat).replace(to_replace, value)._values`?

Reply:
Yes, good point. Added back/updated the test. thx